client-ts/Microsoft.AspNetCore.SignalR.Test.Server/wwwroot/js/hubConnectionTests.js

```diff
@@ -255,16 +255,75 @@ describe('hubConnection', function () {
                     done();
                 });
 
-                hubConnection.start().then(function () {
-                    return hubConnection.invoke('InvokeWithString', message);
-                })
-                .then(function () {
-                    return hubConnection.stop();
+                hubConnection.start()
+                    .then(function () {
+                        return hubConnection.invoke('InvokeWithString', message);
+                    })
+                    .then(function () {
+                        return hubConnection.stop();
+                    })
+                    .catch(function (e) {
+                        fail(e);
+                        done();
+                    });
+            });
+
+            it('can receive server calls without rebinding handler when restarted', function (done) {
+                var options = {
+                    transport: transportType,
+                    protocol: protocol,
+                    logging: signalR.LogLevel.Trace
+                };
+                var hubConnection = new signalR.HubConnection(TESTHUBENDPOINT_URL, options);
+
+                var message = '你好 SignalR！';
```

Comment:
Thought: this and some others could be `const`s, but I'm not going to block any PR